job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/Constants.groovy

```diff
@@ -18,6 +18,7 @@ package org.kie.jenkins.jobdsl
 
 class Constants {
     static final String MAVEN_VERSION = '3.8.1'
+    static final String MAVEN_TOOL = "kie-maven-${MAVEN_VERSION}"
     static final String UPSTREAM_BUILD_MAVEN_VERSION = '3.8.1'
     static final String JDK_VERSION = 'kie-jdk11'
```

Comment:
shouldn't it be like
```suggestion
    static final String JDK_VERSION = '11'
    static final String JDK_TOOL = "kie-jdk${JDK_VERSION}"
```
in order to keep maven and java aligned? :thinking:

Reply:
DONE.
All JAVA_VERSION were changed to JAVA_TOOL